e2e/_suites/ingest-manager/features/ingest-manager.feature

```diff
@@ -0,0 +1,46 @@
+@ingest
+Feature: Enable Fleet user and create initial Kibana setup
+
+Scenario: Enrolling an agent
+  Given there is a "Fleet" user in Kibana
+    And the "Fleet" Kibana setup has been created
+  When the agent binary is installed in the target host
+  Then the dashboards for the agent are present in Elasticsearch
+    And the agent shows up in Kibana
+
+Scenario: Un-enrolling an agent
+  Given there is a "Fleet" user in Kibana
+    And the "Fleet" Kibana setup has been created
+    And the agent binary is installed in the target host
+  When the agent is un-enrolled from Kibana
+  Then no new data shows up in Elasticsearc locations using the enrollment token
```

Comment:
I added `using the enrollment token` to match an existing step below. Is this assumption correct?

Reply:
here I think you should say using the `access token` when an agent enroll into fleet we exchange an enrollmont token for an access token (that is one per agent). 
One you invalidate an enrollment token, the agent already enrolled should continue to work, but you cannot enroll more agents with that enrollment token

Reply:
Thanks for the clarification Nicholas! Please look at L27:33 There is specific scenario for revoking the enrollment token for an agent. Is that what you mean?

Reply:
Mmm, reading your comment, I'd rephrase this second scenario (the one revoking the token) to this:

```gherkin
Scenario: Revoking the enrollment token for an agent
  Given there is a "Fleet" user in Kibana
    And the "Fleet" Kibana setup has been created
    And the agent binary is installed in the target host
    And the agent is un-enrolled from Kibana
  When the enrollment token is revoked
  Then no new data shows up in Elasticsearc locations using the enrollment token
    And the enrolled agent continues to work
```

And I'd create another use case:
```gherkin
Scenario: A revoked enrollment token cannot enroll more agents
  Given there is an enrollment token
  When the enrollment token is revoked
  Then it's not possible to use the token to enroll more agents
```

Does it make sense to you?

Reply:
BTW, we should clarify what `the enrolled agent continues to work` means: i.e. it sends data to elasticsearch, there is an endpoint we can query, a process is running in the host, etc.

Reply:
Combining above two scenarios into one:

```gherkin
Scenario: Revoking the enrollment token for an agent
  Given there is an agent enrolled with an enrollment token
  When the enrollment token is revoked
  Then it's not possible to use the token to enroll more agents
    And the enrolled agent continues to work
```

Reply:
I like this scenario, because it's very straight-forward and simple at the same time. I'd replace what we had. wdyt about rephrasing the `Given...` to `Given an agent is enrolled`? Or do we want to make it clear for this scenario that we need the fleet user and the existence of a valid token?

Reply:
BTW, in what state would be the existing agent? Will it pause? will it continue to send data?